bool blocking
bool checkStatus
---
Mission mission